index.html

```diff
@@ -0,0 +1,198 @@
+                <a href="#" class="hero__btn">Download <span>free</span>!</a>
+            </div>
+            <div class="hero__img-box">
+                <img class="hero__img" src="./images/screen.png" alt="hero image">
+            </div>
+        </section>
+    </header>
+
+    <main>
+        <section class="features" id="features">
+            <h2 class="features__header container">Features</h2>
+            <div class="features-box container">
+                <ul class="feature__item">
+                    <li class="feature__icon"><img src="./images/responsive.svg" alt="mobile phone logo"></li>
+                    <li class="feature__name">fully responsive</li>
+                    <li class="feature__description">Looks amazing on any device: smartphone, tablet, laptop and
+                        desktop.</li>
+                </ul>
+                <ul class="feature__item">
+                    <li class="feature__icon"><img src="./images/customizable.svg" alt="tools logo"></li>
+                    <li class="feature__name">customizable</li>
+                    <li class="feature__description">Change the colors, pictures or any of the sections to suit your
+                        needs.</li>
+                </ul>
+                <ul class="feature__item">
+                    <li class="feature__icon"><img src="./images/design.svg" alt="prize medal logo"></li>
+                    <li class="feature__name">slick and beautiful design</li>
+                    <li class="feature__description">Trendy and fresh design, fits any website.</li>
+                </ul>
+            </div>
+
+        </section>
+
+        <section class="featured-in">
+            <div class="featured-in__box container">
+                <img class="featured-in__logos" src="./images/logos.png" alt="logos of companies we work with">
+            </div>
+        </section>
+
+        <section class="about">
+            <img class="about__photo" src="./images/imac.png" alt="picture of imac monitor">
+            <article class="about-box">
+                <h2 class="about__header">Simple and beautiful</h2>
+                <p class="about__text">Use Nova theme for your next web project. It is completely customizable so you
+                    can change any of the
+                    sections to fit
+                    your needs. Nova Theme is Free for any kind of use, personal and commercial. Have fun and good luck!
+                </p>
+            </article>
+        </section>
+
+        <section class="pricing" id="pricing">
+            <div class="pricing__headers container">
+                <h2 class="pricing__header">Choose your pricing plan</h2>
+                <h3 class="pricing__subheader">Pick any of our super affordable pricing plans</h3>
+            </div>
+            <div class="pricing-box container">
+                <div class="pricing-box__item">
```

Comment:
Ten element też mógłby być listą bo jest to "lista" pakietów cenowych, lub alternatywnie `pricing-box__item` zrobiłbym jako `article` bo to jedna całość tj. jeden pakiet.